### PR TITLE
New Docs

### PR DESCRIPTION
**Pull request by jordancde**

### CODE REVIEW

File: pricing.mdx [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 title: "Pricing"
 sidebarTitle: "Pricing"
-description: "Get to know your pricing per minute based on the providers you choose."
+description: "We only charge for what our customers use. You pay by the minute based on the providers of your choice."
 ---
 
-The vast majority of the cost of using Vapi is the cost of the underlying providers. We charge a base fee on top of the provider cost to cover our costs and to be a sustainable business.
+We have strived to make our pricing as straightforward as possible. Most of what you pay to use the Vapi platform consists of what the Transcription, Model, and Voice providers cost. This will vary based on which providers you choose in the [Dasboard](https://dashboard.vapi.ai). We only charge a base fee on top of the provider costs to run a sustainable business and continue our work in pushing the frontiers of Voice AI.
```
Comment (jordancde) [inline pricing.mdx]: Preference for more blunt / casual language, devs don't like flourish like "strive". Generally good, just that word stuck out

File: pricing.mdx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-When using [Provider Keys](/provider_keys), you'll be billed directly by the provider, not us. When not using Provider Keys, you'll be billed at at cost for those providers through us.
+When using [Provider Keys](/provider_keys), you will be billed directly by the provider. When not using Provider Keys, you will be billed tbrough us at the cost of your chosen providers. Payment info can be entered in the Billing screen of the Dashboard, and occurs automatically based on the number of minutes you use.
```
Comment (nikhilro) [inline pricing.mdx]: through typo

File: pricing.mdx [hunk 3]
```diff
@@ -25,4 +25,4 @@ Based on the providers you select and the keys you provide, you will be automati
   <br /> Deepgram: **\$0.05/min** (=\$3.00/hr) <br /> OpenAI: **\$0.02/min** (=\$1.20/hr) <br /> RimeAI: **\$0.02/min** (=\$1.20/hr)
 </Card>
```
Comment (jordancde) [inline pricing.mdx:35]: Might be good to add some examples here, base fee + Deepgram + GPT-3.5-turbo + rimeAI. = x (cheaper end), base fee + Deepgram + GPT-4-turbo + 11labs (higher end)

**Comment by nikhilro** [comment]
This whole page needs to be a lot simpler. Check out all the Mintlify components here: https://mintlify.com/docs/content/components/accordions. Just play around with them and come up with something good

File: pricing.mdx (hunk 2 above)
Comment (jordancde) [inline pricing.mdx]: spelling, "tbrough"

File: pricing.mdx [hunk 4]
```diff
@@ -1,14 +1,14 @@
 ---
 title: "Pricing"
 sidebarTitle: "Pricing"
-description: "Get to know your pricing per minute based on the providers you choose."
+description: "We only charge for what our customers use. You pay by the minute based on the providers of your choice."
 ---
```
Comment (jordancde) [inline pricing.mdx:7]: I think it'd be good to use https://mintlify.com/docs/content/components/callouts#tip-callouts
to visually highlight the main point about "only pay for what you use"

Need to figure out what to move around to not make it so repetitive.

**Comment by JayC511** [comment]
Created new branch

**Pull request closed without merging**